### ISSUE TITLE
Style: Horizontal space is underutilized on mobile

### ISSUE THREAD

**Issue by fuglede** [scipy/scipy-sphinx-theme-v2#48]
On the main landing pages, the content takes up only about 50% of the available horizontal space on mobile (screenshot below). It's a matter of taste whether this is a good thing or not, but it does cause the content to take up more vertical space than necessary, and it's inconsistent with the subpages which make use of the full screen width (which I would personally prefer).

![Screenshot_20191127-104322](https://user-images.githubusercontent.com/6169306/69712804-ba83df00-1103-11ea-9fb9-1b14e6d9154e.png)

**Comment by mdhaber** [comment]
I noticed this, too, and agree that it's inconsistent with the subpages. Horizontal space is quite limited on mobile already, so I would suggest not restricting it so much further. On the other hand, I can see arguments for a small number of words per line for top-level information. If it was a conscious decision, I wouldn't argue against it.

**Comment by Shekharrajak** [comment]
Yes, the main intention is to make it full screen and most of the pages are full screen . e.g. 
<img width="347" alt="Screenshot 2019-11-30 at 8 23 39 PM" src="https://user-images.githubusercontent.com/5774448/69902075-6735b900-13af-11ea-9afd-4b9d424989ab.png">

I also see similar issue and had put this in lower priority issue for know but it should be fixed.
